config/manager/manager.yaml

```diff
@@ -87,6 +87,10 @@ spec:
               value: registry.redhat.io/openshift-sandboxed-containers/osc-podvm-builder-rhel9@sha256:d635d2e688982986f6942b19948e0a874b093957045335c1009f0765aa9bd42e
             - name: RELATED_IMAGE_PODVM_PAYLOAD
               value: registry.redhat.io/openshift-sandboxed-containers/osc-podvm-payload-rhel9@sha256:cacdcb85e4f421b394765fd4632e6accf53ab6dcf78afd285a1a56bc863dc714
+            - name: RELATED_IMAGE_PODVM_OCI
+              value: registry.redhat.io/openshift-sandboxed-containers/osc-podvm-oci-rhel9:1.10.0
+            - name: OSC_SIGNATURE_DATABASE_KEY
+              value: "MIIFITCCAwmgAwIBAgIUPQq9lQi9U3XCw+W7Fr9q5XMRuAkwDQYJKoZIhvcNAQELBQAwIDEeMBwGA1UEAwwVTXkgY3VzdG9tIGNlcnRpZmljYXRlMB4XDTI1MDQyOTA4NDAyNloXDTI1MDUyOTA4NDAyNlowIDEeMBwGA1UEAwwVTXkgY3VzdG9tIGNlcnRpZmljYXRlMIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAmZ8jwAOUmkY8nDtZb+jt8gw59JyTMfrgUJ2Kan2T3wHNASASGVQZqu8GWAec4hz0aBx6tFhv/smV842TybqHayPQNiVvzisERzdTSrjYdIQqM6oQa58C5s9yIbUyiiVny4etFIrsqB14fGKCdz/Yskg3laU2vfIhNiJkukofHDmadPGo8yijzdQXXN3BeCtC5ER1WR029MbegN+LNoRnN28YOGp9YaUsM0ey/L2nGgh3g7hkxtg1pxKMHywR5+4S5lT93YE9fO4iMmhq+9j3BpoAmQ7S0+bged6Qy/MdKZw76RDw8kVbVBA7hiq8obn+TZ+OZ/fXbsG4dlBMgttPK7KGtrd/95oRiLESQ3IHfqm/HXdhiXtMXlbpVBgMmW3zSeA0iBsZiO5n9240ljgY3QebtF7t3WlwBzVQaE1mro0+QUAIOih0ua5guVzGnQ1QFCQgchoPdd01zqUlXLTEIPJop4AQn9y3jV0RjE53d7c5l/F2Oj+HAAiUEeJ3EiFrqK2z8LUqSkLIlw42Ixp57B/O0/yTWyLzhM4AoHH/mhNrIbz9Dc3KohWn8+iiced0jqEnxFw4fxPfUOvNk6ZvFjz4N+WpCDWCrsRzOuS0gJd7pNRGGmJVyc4mk7FbtzgXgI/wBvLOuGhpGa+zlI5cnoLBqMxNN/wHTMkBRkHlBKECAwEAAaNTMFEwHQYDVR0OBBYEFMdJlU45cIgremkk+qeAUpk2b2pPMB8GA1UdIwQYMBaAFMdJlU45cIgremkk+qeAUpk2b2pPMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggIBAA18dX5s4uRsYDZVaglMWjRbCReoRLHy+HDIgz3pp/THpYNDm3ItQDyN7OgFhcfjoHhiA6OwD3BmF20KUD13HZNc01KgWSNLghZZKDy9pcIMM/vM+5WKwbP9QLbQg5MobUmW9nuWDQJ3/3MYzy754jEvzBPipi+d/AA1ml83TmnXDXTE00/ml5yis1t7lqZdNcqlfVxzmyn8E5t0Wlz7UqE7D6gykst+mQhIjs/C4WWuYdGEIzZQVALnSj3iexnJSZZ0S1szFuk3Tx3rqbxlutOipNg79kiREKnaP3L3++IkIY2oOc8gY2LKWSbqLPqtBwq+HrWEfu6WGJAKOUtEBvdOGmQR/di4u7qY+NG8xuGSdWiKLzeUDd6+7ipjWv4Bj9WJq1LLdqCZ8rpwzYKmXorp5w9h75YZLa2zPkP2WeDllEvOuRDmH2upvDGqlxsc2HMAR/2ux2OuzxsJWABx6Zf93xkdkjBDxEWAgEmnsRkjlDNkIhXbf3xI2zyKS4INpBHbQs41vKXJvr15FZ5DsWz0Rus1P8/IYfYYQFQofwUHbsn79vfqvNinSbdHA4K+gtoT+SKSMPKqw2aJ27z46ku3CQFajibY81E+fAq0mzbjnRZqa2gpsznEbQElEooevJrf0UVrDOVWyQSC1PPeMXgX51Ue/cGfw1Z33FEUIHhR"
```

Comment:
Same concern here, iiuc, this cert is expired.

Reply:
it isn't , it's used for Azure, at the previous commented one my intention was to have it as example (and a place to keep it as this is the key we'll be using for now), i can remove the other one.

Reply:
@snir911 it is unclear why we're hard-coding this key... can you provide the full story ?

Reply:
dropped to changes in plan:)